tests/DictTests.elm

```diff
@@ -135,7 +135,10 @@ keysSuite =
                     |> Expect.equalLists [ 0, 1, 2, 3, 4, 5 ]
 
         --
-        , fuzz (Fuzz.list (Fuzz.tuple ( Fuzz.int, Fuzz.char ))) "fromList" <|
+        , fuzz
+            (Fuzz.list (Fuzz.pair Fuzz.int Fuzz.char))
+            "fromList"
+          <|
```

Comment:
oh lord, the changes were actually caused by different elm format versions you and I (3 years ago) are/were using